### PR TITLE
Added blast specific data on contract

### PR DESCRIPTION
**Pull request by GabrielCartier**

### CODE REVIEW

File: src/Echo.sol [hunk 1]
```diff
@@ -15,12 +15,13 @@
     event OfferAccepted(bytes32 indexed offerId);
     event OfferCanceled(bytes32 indexed offerId);
     event OfferExecuted(bytes32 indexed offerId);
-    // @dev For future use...
+    event OfferRedeeemed(bytes32 indexed offerId, address indexed owner);
 
-    uint16 private immutable CHAIN_ID;
+    // @dev For future use...
+    uint256 private immutable CHAIN_ID;
```
[the remark this answers is not in the document]
Comment (0xjohnnycagewins) [reply src/Echo.sol:21]: why is it complaining? According to the [docs](https://docs.soliditylang.org/en/v0.4.25/style-guide.html#constants), it's the right style. Is there another way to define constants?

File: src/EchoBlast.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    constructor(address owner, address blastPointsAddress) Echo(owner) {
+        IBlastPoints(blastPointsAddress).configurePointsOperator(owner);
+        BLAST.configureAutomaticYield();
+        BLAST.configureClaimableGas();
+    }
+
+    function claimGas() external onlyOwner {
+        BLAST.claimMaxGas(address(this), msg.sender);
+    }
```
[the remark this answers is not in the document]
Comment (0xjohnnycagewins) [reply src/EchoBlast.sol:19]: I guess we could return it?

**Review by 0xjohnnycagewins — APPROVED** [review]
looking good!

File: src/EchoBlast.sol (hunk 2 above)
[the remark this answers is not in the document]
Comment (GabrielCartier) [reply src/EchoBlast.sol:19]: I guess 🤷

**Pull request merged**